R/sequences.R

```diff
@@ -10,7 +10,10 @@
 #' @inheritParams valuetype
 #' @param case_insensitive ignore case when matching, if \code{TRUE}
 #' @param count_min minimum frequency of sequences
-#' @param nested collect nested sequences
+#' @param len_max maxium length of sequences
+#' @param nested if true, collect nested sequences
+#' @param ordered if true, use the Blaheta-Johnson method that distinguishs between 
+#'   the order of tokens, and promotes rare sequences. 
 #' @examples 
```

Comment:
Can we add an example with the `nested = TRUE`? Also some tests that use this new argument and test for the expected behaviour?